carsus/io/output.py

```diff
@@ -0,0 +1,25 @@
+from pandas import read_sql_query
+from carsus.model import Atom, AtomWeight, DataSource
+
+
+def read_basic_atom_data(session):
```

Comment:
this is the script that should be mirrored: https://github.com/tardis-sn/tardisatomic/blob/master/scripts/make_kurucz_hdf5